src/cockpit/channels/filesystem.py

```diff
@@ -154,6 +154,7 @@ class FsReplaceChannel(Channel):
     payload = 'fsreplace1'
 
     _path = None
+    _send_acks = None
```

Comment:
Unused -- possible leftover from moving this to `Channel._ack_bytes`?

Reply:
It actually stopped being used when moving to `_ack_frames` :)  Thanks!